doc/man/dune

```diff
@@ -1,7 +1,7 @@
 (rule
   (targets opam.1)
   (deps opam-topics.inc opam-admin-topics.inc)
-  (action (with-stdout-to %{targets} (run %{bin:opam} --help=groff))))
+  (action (with-stdout-to %{targets} (run ../../src/client/opamMain.exe --help=groff))))
```

Comment:
This change breaks the mingw builds - what's it fixing elsewhere?

Reply:
oh.. I thought dune would have dealt with that and replaced the path